goldenmaster/Plugins/TbEnum/Source/TbEnum/Private/TbEnumSettings.cpp

```diff
@@ -15,9 +15,39 @@ See the License for the specific language governing permissions and
 limitations under the License.
 */
 #include "TbEnumSettings.h"
+#include "ApiGearSettings.h"
 
 UTbEnumSettings::UTbEnumSettings(const FObjectInitializer& ObjectInitializer)
 	: Super(ObjectInitializer)
-	, ServiceConnection(ETbEnumConnection(ETbEnumConnection::CONNECTION_LOCAL))
 {
 }
+
+void UTbEnumSettings::PostInitProperties()
+{
+	Super::PostInitProperties();
+
+	// if set to Local backend, we do not check any further
```

Comment:
the comment should say why, more like no configuration needed for local backend, when there is no connection or something like that
also there should be common file with strings like "Local"